src/ray/rpc/rpc_chaos.cc

```diff
@@ -76,35 +76,35 @@ class RpcFailureManager {
       // 25% chance
       num_remaining_failures--;
       return RpcFailure::Request;
-    } else if (rand == 1) {
+    }
+    if (rand == 1) {
```

Comment:
`rand` is a bad naming, which conflicts with global scoped function: https://man7.org/linux/man-pages/man3/rand.3.html